BUILD.md

```diff
@@ -23,6 +23,7 @@ Instructions for building this repository on Linux, Windows, and MacOS.
         - [Notes About the Automatic Option](#notes-about-the-automatic-option)
     - [Generated source code](#generated-source-code)
     - [Build Options](#build-options)
+    - [CCACHE](#ccache)
```

Comment:
?

Reply:
Seems like the table of contents wasn't updated after the various PR's. I have a vscode extension which automatically creates & updates the TOC so it got changed when I modified the file.